### PR TITLE
Fix errors after typescript migration

### LINKED ISSUE #62: Philips Hue not working properly in Webthings 1.0
While it does send the "Off" command, it seems it can never send  the on command. If I change the state of device manually (ie turn on the light manually) it does show as "On" within Webthings, and I can turn it off, but never can I turn it back on via Webthings. I use this adapter to communicate with HABridge, not a real Philips Hue hub. Has always worked very well, until today.

It reports nothing at all in the log.

### PR DESCRIPTION
**Pull request by tim-hellhake**
fixes #62

### CODE REVIEW

File: philips-hue-adapter.js [hunk 1]
```diff
@@ -80,11 +80,11 @@ class PhilipsHueProperty extends Property {
       return Promise.reject('Read-only property');
     }
 
-    if (this.hasOwnProperty('minimum')) {
+    if (typeof minimum === 'number') {
```
Comment (mrstegeman) [inline philips-hue-adapter.js:83]: Shouldn’t it be checking this.minimum?

File: philips-hue-adapter.js [hunk 2]
```diff
@@ -80,11 +80,11 @@ class PhilipsHueProperty extends Property {
...
       value = Math.max(this.minimum, value);
     }
 
-    if (this.hasOwnProperty('maximum')) {
+    if (typeof minimum === 'number') {
```
Comment (mrstegeman) [inline philips-hue-adapter.js:87]: this.maximum

File: philips-hue-adapter.js [hunk 3]
```diff
@@ -130,14 +130,14 @@ class PhilipsHueProperty extends Property {
         const levelProp = this.device.findProperty('level');
 
         if ((colorModeProp && colorModeProp.value === 'color') ||
-            (!colorModeProp && colorProp)) {
+          (!colorModeProp && colorProp)) {
```
Comment (mrstegeman) [inline philips-hue-adapter.js:133]: Not sure what happened with the spacing on these `if` blocks.

File: philips-hue-adapter.js (hunk 1 above)
Comment (tim-hellhake) [reply philips-hue-adapter.js:83]: Done

File: philips-hue-adapter.js (hunk 3 above)
Comment (tim-hellhake) [reply philips-hue-adapter.js]: Done

File: philips-hue-adapter.js (hunk 2 above)
Comment (tim-hellhake) [reply philips-hue-adapter.js]: done

**Review by mrstegeman — APPROVED** [review]

**Pull request merged**